lib/mix/lib/mix/release.ex

```diff
@@ -777,6 +781,9 @@ defmodule Mix.Release do
     end
   end
 
+  @spec strip_beam(binary()) :: {:ok, binary} | {:error, :beam_lib, term}
+  def strip_beam(binary), do: strip_beam(binary, [])
```

Comment:
We can skip this clause. Instead, pass a default to strip_beam/2 below, as suggested by @fertapric!